models/sipnet/Dockerfile

```diff
@@ -4,19 +4,11 @@ ARG IMAGE_VERSION="latest"
 # ----------------------------------------------------------------------
 # BUILD SIPNET BINARY
 # ----------------------------------------------------------------------
-FROM debian:stretch as sipnet-binary
+FROM pecan/models:${IMAGE_VERSION} as model-binary
 
 # Some variables that can be used to set control the docker build
 ARG MODEL_VERSION=git
 
-# install dependencies
-RUN apt-get update \
-    && apt-get install -y \
-       gcc \
-       git \
-       make \
-    && rm -rf /var/lib/apt/lists/*
-
```

Comment:
Am I right these are deleted because they're all already installed on the pecan/models image?

Reply:
correct